packages/x-data-grid/src/tests/rowSelection.DataGrid.test.tsx

```diff
@@ -592,26 +591,27 @@ describe('<DataGrid /> - Row selection', () => {
       expect(getSelectedRowIds()).to.deep.equal([]);
     });
 
-    describe('ripple', () => {
-      clock.withFakeTimers();
-
-      it('should keep only one ripple visible when navigating between checkboxes', async function test() {
-        if (isJSDOM) {
-          // JSDOM doesn't fire "blur" when .focus is called in another element
-          // FIXME Firefox doesn't show any ripple
-          this.skip();
-        }
-        render(<TestDataGridSelection checkboxSelection />);
-        const cell = getCell(1, 1);
-        fireUserEvent.mousePress(cell);
-        fireEvent.keyDown(cell, { key: 'ArrowLeft' });
-        fireEvent.keyDown(getCell(1, 0).querySelector('input')!, { key: 'ArrowUp' });
-        clock.runToLast(); // Wait for transition
-        await flushMicrotasks();
-        expect(document.querySelectorAll('.MuiTouchRipple-rippleVisible')).to.have.length(1);
-      });
-    });
-  });
+  // Skip on everything as this is failing on all environments on ubuntu/CI
```

Comment:
have you considered rewriting it with async syntax, not using fake timers, but trying to rely on DOM assertions, like checking if the new item has a ripple effect and only one DOM element has a ripple effect..? 🤔

Reply:
Yeah, that was the previously commented code. I only in this last commit reverted the ripple logic to the original one. https://github.com/mui/mui-x/pull/15493/commits/815810d1c5c9ab6dbbf720bac47c0da6459337d3